Sources/Stevenson/JiraService.swift

```diff
@@ -96,110 +99,39 @@ extension JiraService {
-                    self.content = content
-                }
-                public init(text: String) {
-                    self.content = [DocContent.paragraph([.text(text)])]
-                }
+        return try container.make(SlowClient.self)
+            .post(fullURL, headers: self.headers, on: container) { request in
+                try request.content.encode(issue)
+                self.logRequest(logMessage, request)
             }
-
-            public struct DocContent: Content {
-                let type: String
-                let attrs: [String: AnyCodable]?
-                let content: [DocContent]?
-                let text: String?
-
-                fileprivate init(type: String, attrs: [String: Any]? = nil, content: [DocContent]? = nil, text: String? = nil) {
-                    self.type = type
-                    self.attrs = attrs.map { $0.mapValues { AnyCodable($0) } }
-                    self.content = content
-                    self.text = text
-                }
-
-                public static func heading(level: Int, title: String) -> DocContent {
-                    return DocContent(type: "heading", attrs: ["level": level], content: [.text(title)])
-                }
-
-                public static func bulletList(items: [ListItem]) -> DocContent {
-                    return DocContent(type: "bulletList", content: items.map { $0.content })
-                }
-
-                public static func paragraph(_ content: [DocContent]) -> DocContent {
-                    return DocContent(type: "paragraph", content: content)
-                }
-
-                public static func inlineCard(baseURL: URL, ticketKey: String) -> DocContent {
-                    let url = "\(baseURL)/browse/\(ticketKey)#icft=\(ticketKey)"
-                    return DocContent(type: "inlineCard", attrs: ["url": url], content: nil)
-                }
-
-                public static func text(_ text: String) -> DocContent {
-                    return DocContent(type: "text", text: text)
-                }
-
-                public static func hardbreak() -> DocContent {
-                    return DocContent(type: "hardBreak")
-                }
-            }
-
-            public struct ListItem {
-                let content: DocContent
-                public init(content: [DocContent]) {
-                    self.content = DocContent(
-                        type: "listItem",
-                        content: [DocContent.paragraph(content)]
-                    )
+            .catchError(.capture())
+            .flatMap { response in
+                self.logResponse(logMessage, response)
+                if response.http.status == .created {
+                    return try response.content
+                        .decode(CreatedIssue.self)
+                } else {
+                    return try response.content
+                        .decode(ServiceError.self)
+                        .thenThrowing { throw $0 }
```

Comment:
Now properly parsing error responses (instead of failing to decode `CreatedIssue` with `KeyNotFound` if response is an error)